tests/parser/NetworkTests.cpp

```diff
@@ -68,6 +75,10 @@ BOOST_AUTO_TEST_CASE(Branch) {
 
 BOOST_AUTO_TEST_CASE(INVALID_DOWNTREE_NODE) {
```

Comment:
I am not sure what this and the next was supposed to test, but after the change won't both just throw due to both NETWORK and GRUPNET?

Reply:
> but after the change won't both just throw due to both `NETWORK` and `GRUPNET`?

I don't see `GRUPNET` here.  There is `GRUPTREE` but no `GRUPNET` as far as I can tell.

Reply:
My mistake, thanks for pointing it out.